carbon-design-system/carbon-addons-iot-react
# fix(list): indeterminate selection in story

Closes #3148

**Summary**
Fix in List story to make sure parent rows of indeterminate state are now rendered as selected

**Change List (commits, features, bugs, etc)**
- updated story

**Acceptance Test (how to verify the PR)**

1. Go to https://deploy-preview-3160--carbon-addons-iot-react.netlify.app?path=/story/1-watson-iot-list--with-checkbox-multi-selection-and-hierarchy
2. Select a parent row (New York Yankees), then expand it
3. Deselect a child to create indeterminate parent state
4. Make sure the parent row (New York Yankees) has an indeterminate checkbox state and that the row itself is NOT marked as selected with a blue left border.

**Regression Test (how to make sure this PR doesn't break old functionality)**

n.a.



**Things to look for during review**
